docs/cli/edgedb_project/edgedb_project_init.rst

```diff
@@ -21,6 +21,11 @@ to convert an existing directory to a project directory, connecting the
 existing instance to the project. Typically this tool will prompt for specific
 details about how the project should be setup.
 
+.. note::
+
+    The ``edgedb project init`` command is not intended for use with production
```

Comment:
You _can_ use this with a remote instance if it is linked already. I'm not sure there are any use cases where we recommend doing this though.
```bash
$ edgedb project init --server-instance do-test
No `edgedb.toml` found in `/home/fmoor/src/sample-remote-project` or above
Do you want to initialize a new project? [Y/n]
> Y
Specify the name of EdgeDB instance to use with this project [default: do-test]:
> do-test
Do you want to use existing instance "do-test" for the project? [y/n]
> y
Applying migrations...
Everything is up to date. Revision initial
Project linked
To connect to do-test, run `edgedb`
```

@1st1 should we just say this is "not intended for use with remote instances"?

Reply:
Elvis says this was the intent for adding the ability to link project to instance, so I've removed the note here. Removing the note as well from `unlink` and `info` which are both applicable once a self-hosted production instance is linked, that leaves only `project upgrade` that this note is applicable to under the `project` commands, so I also removed the top-level note (at `docs/cli/edgedb_project/index.rst`) since it only seems to be the case for one command.